auditbeat/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2.1'
+services:
+  beat:
+    build: ${PWD}/.
+    depends_on:
+      - proxy_dep
+    env_file:
+      - ${PWD}/build/test.env
+    environment:
+      - KIBANA_HOST=kibana
+      - KIBANA_PORT=5601
+    working_dir: /go/src/github.com/elastic/beats/auditbeat
+    volumes:
+      - ${PWD}/..:/go/src/github.com/elastic/beats/
+    command: make
+
+  # This is a proxy used to block beats until all services are healthy.
+  # See: https://github.com/docker/compose/issues/4369
+  proxy_dep:
+    image: busybox
+    depends_on:
+      elasticsearch: { condition: service_healthy }
+      kibana:        { condition: service_healthy }
+
+  elasticsearch:
+    extends:
+      file: ../testing/environments/${TESTING_ENVIRONMENT}.yml
```

Comment:
If you follow this file you will find that it references `Dockerfile-snapshot` which no longer exists.

https://github.com/elastic/beats/blob/f3de72ec87c0980010eefb79c37489ea790ba118/testing/environments/snapshot-noxpack.yml#L10-L11

Reply:
Strange, metricbeat depends on the same one and still works. Change.

Reply:
@andrewkroh I think I just found a potential issue. Metricbeat has it's own elasticsearch instance for the testing of the module. This is reused also for the tests where data is sent to ES like the template loading.

It could now be argued that this a good thing as template loading etc. is tested with a stable version and it's up to libbeat to test its own functionality against master as each beat just uses the provided functionality. At the same time this was not intended.

The above explains how the noxpack file could have become outdated without breaking the tests.